community/elasticsearch/APKBUILD

```diff
@@ -47,23 +32,22 @@ package() {
 
 	cd "$builddir"
 
-	install -dm755 "$destdir"/lib "$destdir"/modules "$destdir"/bin
-	install -m644 -t "$destdir"/lib lib/* || return 1
+	install -dm755 "$destdir"/lib "$destdir"/bin
+	install -m644 -t "$destdir"/lib lib/*
 
 	install -dm755 "$docdir"
 	install -dm755 "$confdir"
-	install -m644 -t "$confdir" config/* || return 1
+	install -m644 -t "$confdir" config/*
 
 	# remove windows files
 	find bin -type f -name *.bat -exec rm -f {} \;
 	find bin -type f -name *.exe -exec rm -f {} \;
 
 	# ES bin script parses the new jvm.options file
-	install -m755 -t "$destdir"/bin bin/* || return 1
+	install -m755 -t "$destdir"/bin bin/*
 
-	# ES does not run without a transport module
-	install -dm755 "$destdir/modules/$default_module"
-	install -m644 -t "$destdir/modules/$default_module" modules/"$default_module"/* || return 1
+	# Modules are no longer subpkgs
+	cp -rf modules $destdir/
```

Comment:
You have misinterpreted him. He was just against the method you’ve used, not against subpackages.